experimental/PropertyDDS/services/property-query/test/unit/shared/leaf_splitting_and_merging.spec.js

```diff
@@ -45,930 +45,933 @@ function getBtreeParameters(settings) {
   };
 }
 
-describe('Node', function() {
-  let randomGuids = [];
-  const numRandomTests = 30;
-  let summedTime = 0;
-
-  before(() => {
-    let outerRandom = new DeterministicRandomGenerator('fcfaa9c7-8483-85ca-04ee-c20458f86532');
-    for (let i = 0; i < numRandomTests; i++) {
-      randomGuids.push(generateDeterministicGuid(outerRandom));
-    }
-
-    PropertyFactory.register({
-      inherits: 'NamedProperty',
-      typeid: 'adsk.test:TestProperty-1.0.0',
-      properties: [
-        {id: 'name', typeid: 'String'},
-        {id: 'data', typeid: 'NodeProperty'}
-      ]
+const randomGuids = [];
+const numRandomTests = 30;
+const outerRandom = new DeterministicRandomGenerator('fcfaa9c7-8483-85ca-04ee-c20458f86532');
+for (let i = 0; i < numRandomTests; i++) {
+  randomGuids.push(generateDeterministicGuid(outerRandom));
+}
```

Comment:
I lifted these variables out of the `before(() => {..})` block so that I have access to 'randomGuids[]' for inclusion in the test titles.